### PR TITLE
#7 Pin Terraform versions

### LINKED ISSUE #7: Doesn't work for latest terraform 0.13 & aws provider 3.x
The current files won't work for terraform 13 (the latest version to download as per instructions). Error:

```
Could not find required providers, but found possible alternatives:

  hashicorp/mongodbatlas -> terraform-providers/mongodbatlas

If these suggestions look correct, upgrade your configuration with the
following command:
    terraform 0.13upgrade .
```

Even with 0.12.29 there is an error if it tries to install latest AWS (3.*) due to indices.

This fixed it.

```hcl
provider "aws" {
  region = var.aws_region
  version = "2.70.0"
}

### PR DESCRIPTION
**Pull request by mannykary**
We need to pin the versions as the new version of Terraform v0.13 has breaking changes.

Pin to Terraform v0.12.29, AWS provider version 2.70.0, and Mongo DB Atlas provider version 0.6.3.

Resolves #7

### CODE REVIEW

File: infra/deploy.sh [hunk 1]
```diff
@@ -42,6 +42,8 @@ terraform {
     bucket = "fp-${fp_context}-terraform-state"
     key    = "infrastructure.tfstate"
   }
+
+  required_version = "~> 0.12"
```
Comment (joshmorel) [inline infra/deploy.sh]:
```suggestion
  required_version = "~> 0.12.29"
```

I didn't get the appropriate error (running 0.13.0) without including the patch.

Everything else look good!

From the docs:

> ~>: Allows the specified version, plus newer versions that only increase the most specific segment of the
> specified version number. For example, ~> 0.9 is equivalent to >= 0.9, < 1.0, and ~> 0.8.4, is equivalent to >= 
> 0.8.4, < 0.9. This is usually called the pessimistic constraint operator.

**Review by joshmorel — CHANGES_REQUESTED** [review]

**Review by joshmorel — APPROVED** [review]
Great! (tested w both 0.12.29 & 0.13.0)

**Review by ahmedasmar — APPROVED** [review]
Everything looks fine, Thanks Manny

**Comment by royki** [comment]
I tested using both `Terraform(v0.12.29)` & `Terraform(v0.13.0)`. Looks good. Thanks

**Pull request merged**